src/Apps/MyCollection/Routes/EditArtwork/Components/MyCollectionArtworkFormArtistStep.tsx

```diff
@@ -27,9 +62,70 @@ export const MyCollectionArtworkFormArtistStep: React.FC<MyCollectionArtworkForm
 
       <Spacer y={4} />
 
-      {/* TODO: Implement screen */}
+      <Text variant={"lg"}>Select an Artist</Text>
+
+      <Spacer y={4} />
+
+      <ArtistAutoComplete
+        onError={onError}
+        onChange={value => {
+          setQuery(value)
+          console.log(value)
+        }}
+        onSelect={onSelect}
+        placeholder="Search for artists on Artsy"
+        title={""}
```

Comment:
**thought:** is this required 👀

Reply:
Looks weird, I changed the component to not have a title per default 👍